docs/CONFIGURATION-AS-CODE.md

```diff
@@ -35,20 +35,20 @@
 | cloudStatusIntervalSec     | int     | no, default ```10```                     ||
 | noDelayProvision           | boolean | no, default ```false```                  ||
 
-## EC2FleetLabelCloud
+## FleetLabelCloud
 
 More about this type [here](LABEL-BASED-CONFIGURATION.md)
 
-[Definition](https://github.com/jenkinsci/ec2-fleet-plugin/blob/master/src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetLabelCloud.java#L123-L145)
+[Definition](https://github.com/jenkinsci/ec2-fleet-plugin/blob/master/src/main/java/com/amazon/jenkins/ec2fleet/FleetLabelCloud.java#L123-L145)
 
 ## Examples
 
-### EC2FleetCloud (min set of properties)
+### FleetCloud (min set of properties)
 
 ```yaml
 jenkins:
   clouds:
-    - ec2Fleet:
+    - fleet:
```

Comment:
This is a backwards incompatible change. I believe the name `ec2Fleet` comes from the package name. But I might be wrong. 
Do you know if the yaml structure for CasC changed with this commit? (you can test by checking out your commits in this PR -> in the UI create [a new Jenkins cloud ](https://github.com/jenkinsci/ec2-fleet-plugin/blob/master/docs/CONFIGURATION-AS-CODE.md#ec2fleetcloud-min-set-of-properties) using Configuration as Code under `Manage Jenkins` (if I remember right))

Reply:
Also, once you fix the resource names in `EC2FleetCloudConfigurationAsCodeTest` and your tests pass without chagnes to the CasC resource files, that is also a good test to confirm that this commit hasn't made a backwards incompatible change for CasC.

Reply:
@GavinBurris42 Adding `@Symbol("ec2Fleet")` [here](https://github.com/jenkinsci/ec2-fleet-plugin/blob/master/src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java#L1059) should help fix this backwards incompatible change.